modules/machineset-yaml-ibm-power-vs.adoc

```diff
@@ -0,0 +1,73 @@
+// Module included in the following assemblies:
+//
+// * machine_management/creating_machinesets/creating-machineset-ibm-power-vs.adoc
+
+:_content-type: REFERENCE
+[id="machineset-yaml-ibm-power-vs_{context}"]
+= Sample YAML for a compute machine set custom resource on {ibmpowerProductName} Virtual Server
+
+This sample YAML file defines a compute machine set that runs in a specified {ibmpowerProductName} Virtual Server zone in a region and creates nodes that are labeled with `node-role.kubernetes.io/<role>: ""`.
```

Comment:
```suggestion
This sample YAML file defines a compute machine set that runs in a specified {ibmpowerProductName} Virtual Server zone in a specified region and creates nodes that are labeled with `node-role.kubernetes.io/<role>: ""`.
```

are zone and region the same thing? Wondering if we might just be able to say ". . . in a specified  {ibmpowerProductName} zone" or ". . . in a specified  {ibmpowerProductName} region"

Reply:
We have region as well as zone as shown below.
```
ibmcloud_region     = "mon"
ibmcloud_zone       = "mon01"
```
Reference : https://cloud.ibm.com/docs/overview?topic=overview-locations

![image](https://github.com/openshift/openshift-docs/assets/32256846/c348e7ec-b7bb-41cd-a89a-a5526dbdf3f8)

so, here we are referring to a specified zone in a region ( above screenshot provides an example ).
fyi : @stevsmit

Reply:
Sounds good to me. Thanks for clarifying.